### DISCUSSION TITLE
Is it possible to create a new user along with a relation?

### DISCUSSION THREAD

**Discussion by abrman** [pocketbase/pocketbase#1585] [Q&A] [answered]
Is it possible to create a new user with some a automatically created and added relation. 

Example:
```js
await pb.collection("users").create({
  email: email,
  emailVisibility: false,
  password: password,
  passwordConfirm: password,
  notes: [{ text: `My first note!` }],
});
```

**Comment by ganigeorgiev** [comment, marked as answer]
Bulk and transactional create/update/delete is in my todo and eventually will be handled as part of #48 but currently is not supported in the web APIs and you'll have to send individual requests for each of your collections.

If you [using PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) you can attach to the [`OnRecordAfterCreateRequest` hook](https://pocketbase.io/docs/event-hooks/#record-api-hooks) and manually execute a transaction (eg. `app.Dao().RunInTransaction(...)`) to insert your "notes" record and update the created `user.notes` field using the [records](https://pocketbase.io/docs/record-methods/) and [db](https://pocketbase.io/docs/database/) helpers.
